docs/integrations/app-integrations/web-crawler.mdx

```diff
@@ -26,12 +24,19 @@ Here is how to initialize a web crawler:
 CREATE DATABASE my_web 
 WITH ENGINE = 'web';
 ```
+<Tip>
+The above query creates a database called `my_web`. This database by default have a table called `crawler` that we can use to crawl data from a given url/urls.
+</Tip>
 
 ## Usage
 
+<Note>
+Specifying a query `LIMIT` is required. To crawl all pages on a site, consider setting the limit to a high value, such as 10,000, which exceeds the expected number of pages. Be aware that setting a higher limit may result in longer response times.
+</Note>
+
 ### Get Websites Content
```

Comment:
'Get Website Content' is more grammatically accurate, I feel.

Reply:
Since they can use the IN to query multiple websites I think we can leave it as it is